google-cloud-datastore/src/main/java/com/google/cloud/datastore/Transaction.java

```diff
@@ -176,6 +176,205 @@
   <T> QueryResults<T> run(Query<T> query);
 
   /**
+   * Datastore add operation. This method will also allocate id for any entity with an incomplete
+   * key. As opposed to {@link #add(FullEntity)} and {@link #add(FullEntity...)}, this method will
+   * defer any necessary id allocation to commit time.
```

Comment:
I feel like your proposed change alters the meaning of the sentence.

"unlike {@link #add(FullEntity)} and {@link #add(FullEntity...)}" must refer to the second part of the sentence (i.e. when id allocation is made), not to the first part.